### ISSUE TITLE
Can the opening and closing times of the candles be a problem?

### ISSUE THREAD

**Issue by SARPSARP090** [JKorf/Binance.Net#879]
Hello there. I am pulling the information of 10 backward candles in a 4-hour period. However, while the opening and closing times of the candles should coincide with odd numbered hours, they are listed with even numbered hours. I activate the automatic timestamp equalizer for this, but the problem persists.

 ```
var callResult = await client.Spot.Market.GetKlinesAsync("BTCUSDT", KlineInterval.FourHour, limit: 10);
            
            if (callResult.Success)
            {
                var zaman= await client.Spot.System.GetServerTimeAsync(true);
                listBox1.Items.Clear();
                foreach (var veri in callResult.Data)
                {
                   
                    listBox1.Items.Add(veri.Open + "--" + veri.OpenTime);
                }
            }
            else
            {
                listBox1.Items.Add("Hata Oluştu !");
            }

```

**Comment by JKorf** [comment]
I'm not sure I understand the problem. Could you explain a bit more? Why would you expect odd numbered hours?

**Comment by SARPSARP090** [comment]
Because it is applied as 03.00 at night as the closing of the day on the binance exchange. Therefore, 4-hour candle openings should continue as 03.00-07.00, 07.00 - 11.00, 11.00-15.00. However, in the algorithm I used in the example above, a data continues as 00.00 - 04.00, 04.00 - 08.00 - 08.00 - 12.00. How can I get the data to come as 03.00-07.00, 07.00-11.00?

**Comment by SARPSARP090** [comment]
I noticed something. Although the 4-hour bar opening and closing hours are according to the even-numbered hours, when I examined the data, I realized that it was compatible with the 4-hour closings on the binance site. Actually, I don't know why, but I realized that this bar actually belongs to 23.00 even though it looks like there is one at 00:00 on my client. So there is no problem anymore. Thank you for your interest. So we can close the issue.
